string name
int8 plastictype
